custom/conf/app.example.ini

```diff
@@ -258,6 +258,10 @@ CUSTOM_URL_SCHEMES =
 ; Separate the extensions with a comma. To render files without any extension as markdown, just put a comma
 FILE_EXTENSIONS = .md,.markdown,.mdown,.mkd
 
+[svg_render]
```

Comment:
Could this be grouped with other ui settings (eg. `[ui.svg]`)?

Reply:
Another note, could the setting itself be `ENABLE_RENDER`?

Reply:
Once the above have been modified, could you also add the additional settings to the "cheat sheet" found in docs/ directory.